tests/tools/kopiarunner/kopia_snapshotter.go

```diff
@@ -11,6 +12,21 @@ import (
 
 var _ snap.Snapshotter = &KopiaSnapshotter{}
 
+const (
+	contentCacheSizeMBFlag  = "--content-cache-size-mb"
+	metadataCacheSizeMBFlag = "--metadata-cache-size-mb"
+	noCheckForUpdatesFlag   = "--no-check-for-updates"
+	noProgressFlag          = "--no-progress"
+	parallelFlag            = "--parallel"
+)
+
+// Flag value settings.
+const (
+	contentCacheSizeSettingMB  = 500
```

Comment:
nit: can you put those values in the previous const() section next to the flag names?

Reply:
Good call; done ✔️